gulpfile.js

```diff
@@ -33,7 +33,7 @@ gulp.task('lint', function() {
 
 gulp.task('plugin:create', function(){
   if(flags.n && flags.n !== ''){
-    var src = flags.m?'./TEMPLATE-MIN':'./TEMPLATE';
+    var src = flags.m?'./scripts/templates/wrap-min.tmpl':'./scripts/templates/wrap.tmpl';
```

Comment:
Following the pattern [Ionic rc.0](https://github.com/driftyco/ionic/tree/master/scripts/templates/provider) 😉